ionic2/app/pages/about/about.page.scss

```diff
@@ -1,3 +1,7 @@
 .about {
 
 }
+
+ion-item a{
+  text-decoration: none;
```

Comment:
Did you test this @johartl, I might be out of the loop and have not checked but I assumed ionic would do the angular 2 thing and scope this. Otherwise we should move this to the styles in the component/page decorator